packages/web-components/src/components/va-minimal-header/va-minimal-header.tsx

```diff
@@ -27,7 +27,7 @@ export class VaMinimalHeader {
         <div class="va-header">
 
           <a href="/" >
-            <img class="va-logo" src={vaSeal} aria-hidden="true"/>
+            <img class="va-logo" src={vaSeal} alt="VA.gov home"/>
```

Comment:
Some a11y guidance about logos recommends adding the text "logo" to the alt text. Should we do that here?

https://sc.edu/about/offices_and_divisions/digital-accessibility/guides_tutorials/best_practices/alternative_text/logo-alt-text/index.php#:~:text=Using%20the%20word%20%22logo%22%20helps,Follow%20the%20standard%20format.

Reply:
Maybe even better, we can just replicate what is currectly on VA.gov:

Header and Footer logo alt text: `VA logo and Seal, U.S. Department of Veterans Affairs`